build.gradle

```diff
@@ -38,7 +38,7 @@ dependencies {
 
 sourceCompatibility = '1.7'
 group = 'org.opencadc'
-version = '1.1.14'
+version = '1.1.15'
```

Comment:
The master branch was in heavy development the past couple of days.  Please pull the master in and merge it.